service/src/integrationTest/resources/fixtures/case.json

```diff
@@ -7,7 +7,28 @@
         "orderType": [
           "EMERGENCY_PROTECTION_ORDER"
         ]
-      }
+      },
```

Comment:
I would much prefer this in callbackRequest.json and then call it using thre static callbackRequest() method. Thoughts?

Reply:
Makes sense. Change made.